src/EventHub.Domain/Events/Speaker.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Volo.Abp.Domain.Entities;
+
+namespace EventHub.Events
+{
+    public class Speaker : Entity
+    {
+        public Guid SessionId { get; private set; }
+        
+        public Guid UserId { get; private set; }
+
+        public Speaker(Guid sessionId, Guid userId)
```

Comment:
Also add a private constructor for deserialization (while it seems EF Core can handle it, it is better to do that).